
<p><b>Fall 2015</b></p>

<p><b>Technologies:</b> MATLAB, Python, Keras, scikit-learn <br>
<b>Key Words:</b> single-cell, ADMM, robust PCA, deep autoencoder, gene network </p>

<p>I recently had the pleasure of collaborating with <a href="http://www-scf.usc.edu/~ntranos/">Vasilis Ntranos</a>, <a href="http://web.stanford.edu/~gkamath/">Govinda Kamath</a>, <a href="https://math.berkeley.edu/~lpachter/">Lior Pachter</a>, and my advisor <a href="https://web.stanford.edu/~dntse/">David Tse</a> in developing a novel approach for clustering the cells in single-cell RNA-Seq datasets. Inspired by Lior's group's recently developed tool, <a href="https://github.com/pachterlab/kallisto">kallisto</a>, Vasilis, Govinda, and I generated feature vectors for cells based on transcript-compatibility counts rather than quantified gene abundances. The latter requires read alignment, a computationally costly approach especially as the number of reads scale with the increasing number of cells involved in single-cell RNA-Seq experiments. The former requires only pseudoalignment, or a very fast assignment of each read to a set of potential transcripts the read can come from using a k-mer hashing approach.</p>
    
<p>We showed that clustering on TCCs is both a fast and potentially universal approach by reanalyzing two recently published but quite different single-cell datasets. The work was submitted to Genome Biology's Single Cell Omics Special Issue, and our analyses and method are publicly available on <a href="https://github.com/govinda-kamath/clustering_on_transcript_compatibility_counts">GitHub</a>. The preprint is also available <a href="http://biorxiv.org/content/early/2016/01/15/036863">here</a>. Take a look!</p>

<p><b>Fall 2015</b></p>



<p>After a research collaboration with some members of <a http="https://www.serafimb.org/">Serafim Batzoglou</a>'s group, I was inspired to explore how deep networks could improve the analysis of single-cell data. This project was done for the Stanford artificial intelligence course (CS 221) taught by <a http="https://cs.stanford.edu/~pliang/">Percy Liang</a>. The results of a single-cell experiment can be summarized using a design matrix where each entry represents the expression level of a certain gene in a certain cell. The purpose of the project was to leverage the power of deep neural networks to clean the noisy design matrix such that biologically meaningful clusters could be uncovered using simple unsupervised learning algorithms such as <i>k</i>-means clustering. </p>

<p> The most challenging parts of the project involved solving a non-convex optimization problem using ADMM and building an appropriate network with which to process the data, neither of which I had done before. With the help of a <a http="https://web.stanford.edu/~meisamr/index.html">postdoc</a> in my group, I found a convex relaxation to the problem and derived the ADMM update equations. This was also my first practical experience working with deep networks, and luckily the <a http="https://keras.io/">Keras</a> documentation was fantastic. You can read more about about the project details <a href="/projects/proj-3.pdf">here</a>, and you can check out all files related to the project on <a href="https://github.com/jessemzhang/CS221_final_project">GitHub</a>.</p>

