<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Jesse M. Zhang</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-category">Jesse</div>
<div class="menu-item"><a href="index.html" class="current">About&nbsp;Me</a></div>
<div class="menu-item"><a href="Jesse_Zhang_CV.pdf">CV</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Jesse M. Zhang</h1>
</div>
<table class="imgtable"><tr><td>
<img src="jz.jpg" alt="alt text" width="300px" height="300px" />&nbsp;</td>
<td align="left"><p><a href="http://ee.stanford.edu/">Department of Electrical Engineering</a><br />
<a href="http://stanford.edu/">Stanford University</a><br />
Research Interests: Machine Learning + Genomics<br />
[<a href="https://scholar.google.com/citations?user=4jViXZgAAAAJ">Google Scholar</a>]
[<a href="https://github.com/jessemzhang/">GitHub</a>]
[<a href="https://www.linkedin.com/in/jessemzhang">LinkedIn</a>]</p>
</td></tr></table>
<h2>About Me</h2>
<p>As of June 3, 2019, I am building a startup with two friends, and we're based out of SF. Stay tuned. :)</p>
<p>I recently finished my PhD in electrical engineering at Stanford University. While at Stanford, I was a member of <a href="https://web.stanford.edu/~dntse/">David Tse</a>'s group, and my research revolved around analysis of single-cell RNA-Seq datasets and understanding deep learning models. During the winter 2016 and spring 2018 quarters, I helped design and teach a new Stanford course on <a href="http://data-science-sequencing.github.io/">data science for high-throughput sequencing</a>. I have also had the privilege of working for some wonderful organizations during my graduate and undergraduate careers including <a href="https://grail.com/">Grail</a>, <a href="http://www.bd.com/en-us/offerings/capabilities/genomics">Cellular Research</a>, <a href="https://www.mc10inc.com/">MC10</a>, <a href="https://ll.mit.edu/">MIT Lincoln Laboratory</a>, and <a href="http://www.dana-farber.org/">Dana-Farber</a>. Please see my <a href="http://stanford.edu/~jessez/Jesse_Zhang_CV.pdf">CV</a> for more details.</p>
<p>[<a href="https://www.dropbox.com/s/d58t51iwgqrvnwk/Stanford_PhD_Thesis.pdf?dl=0">Dissertation</a>][<a href="https://www.dropbox.com/s/yo3dzl21q7bjq0e/Stanford_PhD_Defense.MOV?dl=0">Defense</a>]</p>
<h2>Publications</h2>
<p><a href="https://arxiv.org/abs/1811.07457">Generalizable Adversarial Training via Spectral Normalization</a><br />
Farzan Farnia*, <b>Jesse M. Zhang</b>*, David N. Tse<br />
*equal contributors<br />
<b>International Conference on Learning Representations, 2019</b><br />
[<a href="https://github.com/jessemzhang/dl_spectral_normalization">Code</a>][<a href="https://www.dropbox.com/s/uswpaw8kucvg822/ICLR_2019_poster_final_2.pdf?dl=0">Poster</a>]</p>
<p><a href="https://www.biorxiv.org/content/10.1101/463265v2">Valid post-clustering differential analysis for single-cell RNA-Seq</a><br />
<b>Jesse M. Zhang</b>, Govinda M. Kamath,  David N. Tse<br />
<b>Research in Computational Molecular Biology, 2019</b><br />
[<a href="https://github.com/jessemzhang/tn_test">Code</a>][<a href="https://docs.google.com/presentation/d/17MojldL849zDCbRs8P919oLdfjtQWSnN4CwHPD0avbA/edit?usp=sharing">Slides</a>][<a href="https://www.dropbox.com/s/56rh4fk85vbirmv/tn_test_recomb_2019_talk.MOV?dl=0">Talk</a>]</p>
<p><a href="https://arxiv.org/abs/1710.02196">Porcupine Neural Networks: (Almost) All Local Optima are Global</a><br />
Soheil Feizi, Hamid Javadi, <b>Jesse Zhang</b>, David Tse<br />
<b>Advances in Neural Information Processing Systems, 2018</b><br />
[<a href="https://github.com/jessemzhang/porcupine_neural_networks">Code</a>][<a href="https://www.dropbox.com/s/sr711gjl8qw9qyk/NIPS_2018_PNN_v2.pdf?dl=0">Poster</a>]</p>
<p><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2092-7">An Interpretable Framework for Clustering
Single-Cell RNA-Seq Datasets</a><br />
<b>Jesse M. Zhang</b>, Jue Fan, H. Christina Fan, David Rosenfeld, and David N. Tse<br />
<b>BMC Bioinformatics, 2018</b><br />
[<a href="https://github.com/jessemzhang/dendrosplit">Code</a>][<a href="https://www.dropbox.com/s/xinym93bksqd0bp/dendrosplit_poster.pdf?dl=0">Poster</a>]</p>
<p><a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0970-8">Fast and accurate single-cell RNA-seq analysis by clustering of transcript-compatibility counts</a><br />
Vasilis Ntranos*, Govinda M. Kamath*, <b>Jesse M. Zhang</b>*, Lior Pachter and David N. Tse<br />
*equal contributors<br />
<b>Genome Biology, 2016</b><br />
[<a href="https://github.com/govinda-kamath/clustering_on_transcript_compatibility_counts">Code</a>][<a href="https://www.dropbox.com/s/977wqrlq0lyteqq/TCC_poster.pdf?dl=0">Poster</a>]</p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/25482560">Lysine-specific demethylase 1 has dual functions as a major regulator of androgen receptor transcriptional activity</a><br />
Changmeng Cai, Housheng Hansen He, Shuai Gao, Sen Chen, Ziyang Yu, Yanfei Gao, Shaoyong Chen, Mei Wei Chen, <b>Jesse Zhang</b>, Musaddeque Ahmed, Yang Wang, Eric Metzger, Roland Sch√ºle, X. Shirley Liu, Myles Brown, and Steven P. Balk<br />
<b>Cell Reports, 2014</b></p>
<h2>Information</h2>
<ul>
<li><p>Room 260<br />Packard Electrical Engineering Building<br />350 Serra Mall<br />Stanford, CA 94305</p>
</li>
<li><p>Email: jessez@stanford.edu<br /></p>
</li>
<li><p>Personal Website: <a href="https://jessemzhang.com">jessemzhang.com</a></p>
</li>
</ul>
<div id="footer">
<div id="footer-text">
Page generated 2019-10-24 00:33:38 PDT, by <a href="http://jemdoc.jaboc.net/">jemdoc</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
