<p><b>Fall 2015 and Winter 2016</b></p>

<p><b>Technologies:</b> iPython, scikit-learn, networkx, Bash, AWS, genomics tools <br>
<b>Key Words:</b> single-cell RNA-seq, equivalence class, transcript compatibility count, clustering </p>

<p>I recently had the pleasure of collaborating with <a href="http://www-scf.usc.edu/~ntranos/">Vasilis Ntranos</a>, <a href="http://web.stanford.edu/~gkamath/">Govinda Kamath</a>, <a href="https://math.berkeley.edu/~lpachter/">Lior Pachter</a>, and my advisor <a href="https://web.stanford.edu/~dntse/">David Tse</a> in developing a novel approach for clustering the cells in single-cell RNA-Seq datasets. Inspired by Lior's group's recently developed tool, <a href="https://github.com/pachterlab/kallisto">kallisto</a>, Vasilis, Govinda, and I generated feature vectors for cells based on transcript-compatibility counts rather than quantified gene abundances. The latter requires read alignment, a computationally costly approach especially as the number of reads scale with the increasing number of cells involved in single-cell RNA-Seq experiments. The former requires only pseudoalignment, or a very fast assignment of each read to a set of potential transcripts the read can come from using a k-mer hashing approach.</p>

<p>We showed that clustering on TCCs is both a fast and potentially universal approach by reanalyzing two recently published but quite different single-cell datasets. The work was submitted to Genome Biology's Single Cell Omics Special Issue, and our analyses and method are publicly available on <a href="https://github.com/govinda-kamath/clustering_on_transcript_compatibility_counts">GitHub</a>. The final manuscript is published online by Genome Biology and is available <a href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0970-8">here</a>. Take a look!</p>
