%%%%%%%%%%%%%%%%% PREAMBLE %%%%%%%%%%%%%%%%%%%%%%%%%%%%
%Change the font size of your document - 10pt, 12.1pt, etc.
\documentclass[letterpaper,11pt,oneside]{article}
\usepackage[utf8]{inputenc}
\usepackage{setspace}
\usepackage{hyperref}
\usepackage{enumitem}
\usepackage{graphicx}
\usepackage[left=.75in, right=.75in, bottom=0.75 in, top=0.75 in]{geometry}
\usepackage{longtable}


%Changes the page numbers - {arabic}=arabic numerals, {gobble}=no page numbers, {roman}=Roman numerals
\pagenumbering{gobble}

%%%%%%%%%%%%%%%%% END OF PREAMBLE %%%%%%%%%%%%%%%%%%%%%

\begin{document}

%%%%%%%%%%%%%%%%% NAME OF APPLICANT %%%%%%%%%%%%%%%%%%%

\noindent  \LARGE{\textbf{Jesse Zhang}}  \\
\vspace{-2ex}
\hrule
\normalsize

%%%%%%%%%%%%%%%%% CONTACT INFORMATION %%%%%%%%%%%%%%%%%
% Your email address, website, and Skype name are links to send email, open your website and add you on Skype. 

\begin{center}
\begin{tabular}{l l}
 Stanford University    & \hspace{1in} \href{mailto:jessez@stanford.edu}{\url{jessez@stanford.edu}} \\
 Electrical Engineering Department    & \hspace{1in} {\url{stanford.edu/~jessez}}   \\
 Stanford, CA 94305 & \hspace{1in} (857) 636-9152 \\
\end{tabular}
\end{center}


% Education %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\noindent \begin{longtable}{@{} p{2.5cm} p{14.8cm}}
 \large{\textsc{Education}}    & \textbf{Stanford University}, Stanford, CA \hfill \textit{09/2014-Present} \\
     & Ph.D., Electrical Engineering (anticipated 2019) \\
     & Research interests: Machine Learning, Statistics, Genomics \\ 
     & Advisor: David Tse \\
     & \\
     & \textbf{Stanford University}, Stanford, CA \hfill \textit{09/2014-06/2016} \\
     & M.S., Electrical Engineering \\
%     & GPA: 4.02/4.30 \\
     & \\
     & \textbf{Tufts University}, Medford, MA \hfill \textit{09/2010-05/2014} \\
     & B.S., Electrical Engineering \\
%     & GPA: 3.96/4.00 \\
     & \\
     & \textbf{Newton South High School}, Newton, MA \hfill \textit{09/2006-06/2010} \\
     & \\
     & \\
  
     
%% Research %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%  \large{\textsc{Research}}  & \textbf{Spatial mapping of single cells using single-cell RNA-seq} \\
%  \large{\textsc{Projects}} & \textit{Collaboration with R. Chiang, V. Ntranos, M. Snyder, and D. Tse} \\
%& 
%\vspace{-7mm}
%\begin{itemize}[leftmargin=.5cm]
%	\setlength\itemsep{-0.3em}
%	\item Developing computational strategies for estimating 3D spatial orientation of cells using data produced by a novel single-cell RNA-seq approach
%\end{itemize} 
%\\  
%
%& \textbf{Deep models for learning in genomics} \\
%  & \textit{Collaboration with F.Farnia, G.Kamath, and D.Tse} \\
%& 
%\vspace{-7mm}
%\begin{itemize}[leftmargin=.5cm]
%	\setlength\itemsep{-0.3em}
%	\item Designing models for sequence-level prediction of genomic features
%	\item Developing theory for initializing weights in unsupervised deep networks
%\end{itemize} 
%\\
%
%    & \textbf{Single-cell RNA-seq clustering using transcript compatibility counts} \\
%    & \textit{Collaboration with V. Ntranos, G. Kamath, L. Pachter, and D. Tse} \\
%& 
%\vspace{-7mm}
%\begin{itemize}[leftmargin=.5cm]
%	\setlength\itemsep{-0.3em}
%	\item Developed and implemented novel concept of clustering cells from single-cell RNA-seq datasets based on transcript compatibility counts
%	\item Published work in Genome Biologyâ€™s Single Cell Omics Special Issue
%\end{itemize} 
%\\
%
%    & \textbf{Single-cell RNA-seq analysis using deep autoencoders and rPCA} \\
%    & \textit{Collaboration with B. Wang, J. Zhu, and S. Batzoglou} \\
%& 
%\vspace{-7mm}
%\begin{itemize}[leftmargin=.5cm]
%	\setlength\itemsep{-0.3em}
%	\item Developed method for cleaning single-cell RNA-seq data by implementing a modified version of rPCA. Augmented the rPCA objective with a gene-similarity matrix estimated using a deep autoencoder
%\end{itemize} 
%\\
%& \\

     
% Work %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
  \large{\textsc{Work}}  & \textbf{Grail}, Menlo Park, CA \\*
  \large{\textsc{Experience}}  & \textit{Computational Biology Contractor}  \hfill \textit{12/2017-08/2018} \\* 
  & \textit{Computational Biology Intern}   \hfill \textit{08/2017-12/2017} \\*
& 
\vspace{-7mm}
\begin{itemize}[leftmargin=.5cm]
	\setlength\itemsep{-0.3em}
	\item Building classifiers and other machine learning tools using Python and R for analysis of cancer genomics data
\end{itemize} 
\\

& \textbf{Cellular Research}, Menlo Park, CA \\*
& \textit{Bioinformatics Intern} \hfill \textit{06/2016-09/2016} \\* 
& 
\vspace{-7mm}
\begin{itemize}[leftmargin=.5cm]
	\setlength\itemsep{-0.3em}
	\item Worked on ResolveTM system as part of an interdisciplinary team of biologists, engineers, and bioinformaticians
	\item Designed a Python library for automated analysis of high-dimensional single-cell RNA-seq data (clustering and feature selection)
\end{itemize} 
\\

%& \textbf{Stanford MIIL}, Stanford, CA\\
%& \textit{EE PhD rotation student} \hfill \textit{09/2014-12/2014}  \\ 
%& 
%\vspace{-7mm}
%\begin{itemize}[leftmargin=.5cm]
%	\setlength\itemsep{-0.3em}
%	\item Simulated small animal CZT PET system with variable aperture using GATE software
%	\item Created MATLAB algorithms for testing normalization methods on simulated data
%\end{itemize} 
%\\

& \textbf{MC10, Inc.}, Cambridge, MA \\
& \textit{R\&D Intern} \hfill \textit{05/2014-08/2014} \\ 
& 
\vspace{-7mm}
\begin{itemize}[leftmargin=.5cm]
	\setlength\itemsep{-0.3em}
	\item Implemented machine learning and signal processing MATLAB algorithms to facilitate real-time and offline accelerometer data analysis
	\item Collaboratively optimized hardware-software interface
\end{itemize} 
\\

& \textbf{MIT Lincoln Laboratory}, Lexington, MA \\*
& \textit{Electrical Engineering Co-op} for Group 33 \hfill \textit{09/2013-05/2014} \\* 
& \textit{Electrical Engineering Intern} for Group 33 \hfill \textit{06/2013-08/2013} \\*
& 
\vspace{-7mm}
\begin{itemize}[leftmargin=.5cm]
	\setlength\itemsep{-0.3em}
	\item Developed MATLAB algorithms to intelligently extract trace from ionogram images
	\item Created graphical user interface in MATLAB to facilitate ionogram image processing
\end{itemize} 
\\

& \textbf{Tufts Biomedical Engineering Department}, Medford, MA \\*
& \textit{Researcher} under supervision of David Kaplan, Ph.D. \hfill \textit{09/2011-08/2012} \\ *
& 
\vspace{-7mm}
\begin{itemize}[leftmargin=.5cm]
	\setlength\itemsep{-0.3em}
	\item Designed and constructed gold circuits on silk scaffolds using soldering, gold sputter coating, and AutoCAD to control and detect neuronal signals
	\item Processed and analyzed neuronal signals using MATLAB and pCLAMP software
\end{itemize} 
\\

& \textbf{Dana Farber Cancer Institute}, Boston, MA \\
& \textit{Researcher} under supervision of Myles Brown, M.D. \hfill \textit{05/2011-08/2011} \\ 
& 
\vspace{-7mm}
\begin{itemize}[leftmargin=.5cm]
	\setlength\itemsep{-0.3em}
	\item Conducted experiments to define role of lysine-specific demethylase 1 in human hormone dependent and independent prostate cancer
	\item Performed computational analysis of results using MS Excel, python and cistrome.org
\end{itemize} 
\\
& \\

% Teaching %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
  \large{\textsc{Teaching}}  & \textbf{EE 372: Data Science for High-Throughput Sequencing}, Stanford, CA \\
  \large{\textsc{Experience}}  & \textit{Teaching Assistant} \hfill \textit{01/2018-03/2018}  \\ 
  & \textit{Teaching Assistant} \hfill \textit{03/2016-06/2016}  \\ 
& 
\vspace{-7mm}
\begin{itemize}[leftmargin=.5cm]
	\setlength\itemsep{-0.3em}
	\item Worked with academic advisor and fellow group member to design the first course in the Stanford electrical engineering department on computational problems in genomics
	\item Prepared lecture notes, led recitation sections, wrote questions for problem sets, designed and updated a course website: \texttt{data-science-sequencing.github.io}
\end{itemize} 
\\

& \textbf{Stanford Athletic Academic Resource Center}, Stanford, CA \\
& \textit{Tutor} \hfill \textit{03/2016-06/2016} \\ 
& 
\vspace{-7mm}
\begin{itemize}[leftmargin=.5cm]
	\setlength\itemsep{-0.3em}
	\item Tutored probabilistic systems analysis for three Stanford undergraduate athletes
\end{itemize} 
\\

& \textbf{Tufts Academic Resource Center}, Medford, MA \\*
& \textit{Head Tutor} \hfill \textit{08/2013-05/2014}  \\*
& \textit{Resident Head Tutor} \hfill \textit{08/2012-05/2013}  \\*
& 
\vspace{-7mm}
\begin{itemize}[leftmargin=.5cm]
	\setlength\itemsep{-0.3em}
	\item Tutored introductory physics, introductory chemistry, calculus III, differential equations, and linear algebra
	\item Held large-scale review sessions, weekly office hours, 1-on-1 sessions
\end{itemize} 
\\
& \\
     
% Publications %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%    
 \large{\textsc{Papers}} & Feizi, S., Javadi, H., \textbf{Zhang, J. M.}, \& David, N. T. (2017). Porcupine Neural Networks: (Almost) All Local Optima are Global. In \textit{Advances in Neural Information Processing Systems 32}, 2018. \\
 	& \\
 
     & \textbf{Zhang, J. M.}, Fan, J., Fan, H. C., Rosenfeld, D., \&  David, N. T. (2018). An Interpretable Framework for Clustering Single-Cell RNA-Seq Datasets. \textit{BMC Bioinformatics}, 19(1), 93. \\
     & \\
     
     & Ntranos, V.*, Kamath, G. M.*, \textbf{Zhang, J. M.,*} Pachter, L., \& David, N. T. (2016). Fast and accurate single-cell RNA-Seq analysis by clustering of transcript-compatibility counts. \textit{Genome biology}, 17(1), 1. (*equal contributors) \\
     & \\
     
     & Cai, C., He, H. H., Gao, S., Chen, S., Yu, Z., Gao, Y., Chen, S., Chen, M.W., \textbf{Zhang, J.}, Ahmed, M., Wang, Y., Metzger, E., Sch\"{u}le, R., Liu, X. S., Brown, M., \& Balk, S. P. (2014). Lysine-specific demethylase 1 has dual functions as a major regulator of androgen receptor transcriptional activity. \textit{Cell reports}, 9(5), 1618-1627. \\
     & \\ 
     & \\
     
 % Talks  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 

 \large{\textsc{Invited}}  & Tufts University electrical engineering seminar  \hfill \textit{04/2018}\\*
 \large{\textsc{Talks}} & Becton Dickinson seminar  \hfill \textit{06/2016} \\
 & \\
 & \\
    
% Honors and Awards %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 

 \large{\textsc{Honors and}}  & \textbf{National Science Foundation Graduate Fellowship}   \\*
 \large{\textsc{Awards}} & Honorable mention \hfill \textit{03/2016} \\*
& \\*
& \textbf{Tufts University} \\
& Summa Cum Laude \hfill \textit{05/2014} \\
& The Amos Emerson Dolbear Scholarship (\$1355.25) \hfill \textit{04/2014} \\
& The Class of 1898 Prize (\$1983.91) \hfill \textit{04/2014} \\
& Tau Beta Pi \hfill \textit{11/2012} \\
& Eta Kappa Nu \hfill \textit{10/2012} \\
& Howard Sample Prize Scholarship in Physics (\$566.33) \hfill \textit{04/2012} \\
& \\
& \textbf{Chinese Consolidated Benevolent Association of New England} \\
& CCBA Scholarship (\$2500.00) \hfill \textit{12/2010} \\
& \\
& \textbf{Junior Achievement of Northern New England} \\
& Stephen G. Sullivan Scholarship (\$1000.00) \hfill \textit{06/2010} \\
& \\
& \\
     
% Skills %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
  \large{\textsc{Skills}}   & \textbf{Languages:} Python, R, MATLAB, C++, Bash \\
   & \textbf{Packages:} Jupyter, TensorFlow, CVX, NumPy, SciPy, scikit-learn, Git, \LaTeX \\
\end{longtable}

\end{document}

